### DISCUSSION TITLE
Why should I write database entire schema migration?

### DISCUSSION THREAD

**Discussion by onryldrm025** [prisma/prisma#23570] [Q&A]
### Question

I just started Prisma and I found it useful, thank you to everyone who contributed, but I cannot use it in a db with old tables. I get the Drift detected error. There is a solution in the document. Yes, but I do not want to use all schema prisma. I only want to create my schema.prisma models and process them, but I think there is no way to do this. Why do we have to do all schema migration? Can't this be done optionally?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @onryldrm025 👋 

Thank you for raising this question.

>  I get the Drift detected error.

When do you get this error? which command are you invoking?

> I only want to create my schema.prisma models and process them, but I think there is no way to do this.

You can generate `schema.prisma` file automatically by invoking `npx prisma db pull`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
